### ISSUE TITLE
Certain buffs not being transferred by DRG merit ability Empathy

### ISSUE THREAD

**Issue by tacbox3** [EdenServer/community#4868]
### Checklist

- [ x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ x] I have provided my client version in the details. (type /ver into your game window)
- [ x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

The following buffs were known in era to transfer to the wyvern via Empathy, but do not currently transfer on Eden:
- Utsusemi
- Max HP boost (i.e. Monarch's Drink)
- basically all (stat) boost effects (inferred not transferring on Eden, not tested)
- Potency
- Regain

There may be additional buffs not listed here that are also not transferrable on Eden. Here are 2008 wiki entries listing possible buffs for transfer: 
https://www.bg-wiki.com/index.php?title=Empathy&oldid=53140
https://ffxiclopedia.fandom.com/wiki/Empathy?oldid=673045

Note that only defense boost had explicitly been tested in 2008, but this buff is in the same class as the other stat boosts now known to all transfer:
https://ffxiclopedia.fandom.com/wiki/Empathy

**Comment by tacbox3** [comment]
https://www.bluegartr.com/threads/65148-Dragoon-group-2-merits-help

there is also some discussion in this thread about body boost in particular, July 2008

**Comment by 59blargedy** [comment]
resolved (mostly) in April 18, 2022 patch. when you have the time, can you update the op to what is left? off top of head utsu remains

**Comment by Steam-arch** [comment]
Empathy does not transfer a list of buffs atm. Atleast: Berserk, Defender, Most (?) COR rolls (tested nin chaos hunters, war roll), Utsusemi: Ichi/Ni, Hasso, Meditate arent transferred via empathy

Protect, Cocoon, /brd Minuet2 & Madrigal do transfer atleast so would maybe assume brd songs work properly. All 5/5 empathy, less than 5 buffs up

![image](https://user-images.githubusercontent.com/69089249/193659798-35f5b073-5381-4d74-bba4-2416c5074988.png)
![image](https://user-images.githubusercontent.com/69089249/193659621-1b4cf4c0-002d-414f-a3fb-14d801b9da0b.png)
![image](https://user-images.githubusercontent.com/69089249/193660419-862de806-5ba0-42e3-899c-595f4d3048fb.png)
